Extensions/tweaks.js

```diff
@@ -1,6 +1,5 @@
-
 //* TITLE Tweaks **//
-//* VERSION 5.0.1 **//
+//* VERSION 5.0.2 **//
```

Comment:
this is a ``minor`` version bump, not a ``patch``. You are adding new functionality